### PR TITLE
sveltified footer and removed moderators.svelte

### LINKED ISSUE #79: more sveltifying needed ...
```

  <div class="mt-6">
    <a
      href="https://youtube.com/eddiejaoude?sub_confirmation=1"
      class="inline-block text-sm m-2 px-4 py-2 leading-none border rounded border-white text-white hover:border-transparent hover:text-teal-500 hover:bg-white transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-110"
    >
      <i class="fab fa-youtube-square"></i>
      YouTube</a
    >
    <a
      href="http://github.com/EddieJaoudeCommunity"
      class="inline-block text-sm m-2 px-4 py-2 leading-none border rounded border-white text-white hover:border-transparent hover:text-teal-500 hover:bg-white transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-110"
    >
      <i class="fab fa-github-square"></i>
      GitHub</a
    >
    <a
      href="https://discord.com/invite/jZQs6Wu"
      class="inline-block text-sm m-2 px-4 py-2 leading-none border rounded border-white text-white hover:border-transparent hover:text-teal-500 hover:bg-white transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-110"
    >
      <i class="fab fa-discord"></i>
      Discord</a
    >
  </div>
```
This can be made a separate component too what do you say ?

### PR DESCRIPTION
**Pull request by UnevenCoder**
closes #79
Also added footer data to data.js

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -126,3 +126,21 @@ export const moderators = [
...
+  },
+  {
+    text: "GitHub",
+    i: "fab fa-github-square",
+    link: "http://github.com/EddieJaoudeCommunity"
+  },
+  {
+    text: "Discord",
+    i:"fab fa-discord",
+    link:"https://discord.com/invite/jZQs6Wu"
```
Comment (BOLT04) [inline src/data.js:144]:
```suggestion
    i: "fab fa-discord",
    link: "https://discord.com/invite/jZQs6Wu"
```

File: src/Footer.svelte [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    export let link
+</script>
+
+    <a
+      href={link}
+      class="inline-block text-sm m-2 px-4 py-2 leading-none border rounded border-white text-white hover:border-transparent hover:text-teal-500 hover:bg-white transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-110"
+    >
+      <i class={i}></i>
+      {text}</a
+    >
```
Comment (BOLT04) [inline src/Footer.svelte:13]:
```suggestion
<a
  href={link}
  class="inline-block text-sm m-2 px-4 py-2 leading-none border rounded border-white text-white hover:border-transparent hover:text-teal-500 hover:bg-white transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-110"
>
  <i class={i}></i>
  {text}
</a>
```

File: src/Footer.svelte [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<script>
+    export let text
+    export let i 
+    export let link
```
Comment (BOLT04) [inline src/Footer.svelte:4]:
```suggestion
    export let text;
    export let i;
    export let link;
```

File: src/App.svelte [hunk 4]
```diff
@@ -1,7 +1,9 @@
 <script>
   import Card from "./Card.svelte";
   import HorizontalCard from "./HorizontalCard.svelte";
-  import { moderators, projects } from "./data";
+  import Footer from "./Footer.svelte"
```
Comment (BOLT04) [inline src/App.svelte:4]:
```suggestion
  import Footer from "./Footer.svelte";
```

**Review by BOLT04 — COMMENTED** [review]
@UnevenCoder thank you, this looks great 👍. Just left some adjustments but this can be merged

**Review by BOLT04 — APPROVED** [review]

**Comment by BOLT04** [comment]
@eddiejaoude the `cypress-run` action is not running and is blocking the PR 😅, do you know how to fix it?

**Comment by alrifay** [comment]
@BOLT04 the action started when I edited the pull request description

**Comment by UnevenCoder** [comment]
I am sorry was buys with preparing for some tests .. thanks for commuting to it 😅

**Pull request merged**